GreenMile/src/Web/Models/Recipe.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+
+namespace Web.Models
+{
+    public class Recipe
+    {
+        [Required, MaxLength(20, ErrorMessage = "Recipe name too long!")]
+        [Key]
+        public string recipeName { get; set; } = string.Empty;
```

Comment:
Public member names should always be [PascalCase](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions)
```suggestion
        public string RecipeName { get; set; } = string.Empty;
```